File: fastapi/app/core/base/httpCode.py
class HttpCode:
    """HTTP 状态码常量 - 与 HTTP 响应状态码一致，同时作为响应体中的 code 字段值"""

    OK: int = 200
    BAD_REQUEST: int = 400
    UNAUTHORIZED: int = 401
    FORBIDDEN: int = 403
    NOT_FOUND: int = 404
    CONFLICT: int = 409
    UNPROCESSABLE_ENTITY: int = 422
    TOO_MANY_REQUESTS: int = 429
    INTERNAL_SERVER_ERROR: int = 500
    BAD_GATEWAY: int = 502
    SERVICE_UNAVAILABLE: int = 503
    GATEWAY_TIMEOUT: int = 504

File: gateway/src/main/java/com/hcsy/gateway/common/constants/HttpCode.java
package com.hcsy.gateway.common.constants;

/**
 * HTTP 状态码常量
 */
public class HttpCode {
    public static final int OK = 200;
    public static final int BAD_REQUEST = 400;
    public static final int UNAUTHORIZED = 401;
    public static final int FORBIDDEN = 403;
    public static final int NOT_FOUND = 404;
    public static final int CONFLICT = 409;
    public static final int UNPROCESSABLE_ENTITY = 422;
    public static final int TOO_MANY_REQUESTS = 429;
    public static final int INTERNAL_SERVER_ERROR = 500;
    public static final int BAD_GATEWAY = 502;
    public static final int SERVICE_UNAVAILABLE = 503;
    public static final int GATEWAY_TIMEOUT = 504;

    private HttpCode() {}
}
